lib/govuk_index/presenters/expanded_links_presenter.rb

```diff
@@ -81,15 +81,20 @@ def organisation_slugs(type)
     def parts_of_taxonomy(taxon_hash)
       parents = [taxon_hash["content_id"]]
 
-      direct_parents = taxon_hash.dig("links", "parent_taxons")
-      while direct_parents && !direct_parents.empty?
+      parent = taxon_hash
+
+      loop do
```

Comment:
Just a minor comment. It would be easier to read this section if it was obvious what's being looped over. 

It could just be as simple as converting the `loop` back to the original `while` and move
getting the next parent to a function.

e.g.
```
direct_parents = parent_taxon(taxon_hash)
```
and in the `while` call:

```
direct_parents = parent_taxon(direct_parents)
```

and have `parent_taxon` return the result of the or:

```
def parent_taxon(taxon)
  taxon.dig("links", "parent_taxons") ||
    taxon.dig("links", "root_taxon") ||
    []
end
```